src/components/verify/verify-page.tsx

```diff
@@ -100,9 +102,15 @@ export const VerifyPage: React.FunctionComponent = () => {
                 setIssuer(document.issuers.map((issuer) => issuer.identityProof?.location).join(","));
               }
             });
-            Promise.all([promises[DNS_TXT_VERIFIER_INDEX], wait(WAIT)]).then(([fragment]) => {
-              setIssuerStatus(isValid([fragment], ["ISSUER_IDENTITY"]) ? Status.RESOLVED : Status.REJECTED);
-            });
+            Promise.all([promises[DNS_TXT_VERIFIER_INDEX], promises[ALLOWED_ISSUERS_VERIFIER_INDEX], wait(WAIT)]).then(
```

Comment:
Do you want to add in the DNS-DID fragment in here as well?

Reply:
nah I want did as another MR support :+1: